app/celery/tasks.py

```diff
@@ -371,3 +376,43 @@ def process_updates_from_file(response_file):
     NotificationUpdate = namedtuple('NotificationUpdate', ['reference', 'status', 'page_count', 'cost_threshold'])
     notification_updates = [NotificationUpdate(*line.split('|')) for line in response_file.splitlines()]
     return notification_updates
+
+
+@notify_celery.task(bind=True, name="send-inbound-sms", max_retries=5, default_retry_delay=300)
+@statsd(namespace="tasks")
+def send_inbound_sms_to_service(self, inbound_sms_id, service_id):
+    inbound_api = get_service_inbound_api_for_service(service_id=service_id)
+    if not inbound_api:
+        # No API data has been set for this service
+        return
+
+    inbound_sms = dao_get_inbound_sms_by_id(service_id=service_id,
+                                            inbound_id=inbound_sms_id)
+    data = {
+        "id": str(inbound_sms.id),
+        "from_number": inbound_sms.user_number,
+        "content": inbound_sms.content,
+        "date_received": inbound_sms.provider_date.strftime(DATETIME_FORMAT)
+    }
+
+    response = request(
+        method="POST",
+        url=inbound_api.url,
+        data=json.dumps(data),
+        headers={
+            'Content-Type': 'application/json',
+            'Authorization': 'Bearer {}'.format(inbound_api.bearer_token)
+        },
+        timeout=60
+    )
+    try:
+        response.raise_for_status()
+    except HTTPError as e:
+        current_app.logger.exception("Exception raised in send_inbound_sms_to_service for service_id: {} and url: {}. "
+                                     "\n{}".format(service_id, inbound_api.url, e))
+        if e.response.status_code >= 500:
+            self.retry(queue=QueueNames.RETRY,
+                       exc='Unable to send_inbound_sms_to_service for service_id: {} and url: {}. \n{}'.format(
+                           service_id, inbound_api.url, e))
+    except self.MaxRetriesExceededError:
```

Comment:
this won't catch the retry error - this except will only look at `response.raise_for_status`.

eg try running this:

```python
try:
  raise KeyError('foo')
except KeyError:
  raise ValueError('bar')
except ValueError:
  print('hello')
```

You'll need to wrap the `self.retry` call in a separate try catch for the `MaxRetriesExceededError`